### PR TITLE
Added config server example

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
related to spring-cloud/spring-cloud-sleuth#1915

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
cc jonatan-ivanov

**Review by spencergibb — APPROVED** [review]

File: config-server/.mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```
Comment (jonatan-ivanov) [inline config-server/.mvn/wrapper/maven-wrapper.properties:1]: Should we use the latest?

File: config-server/pom.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.example.sleuthsamples</groupId>
+	<artifactId>config-server</artifactId>
+	<version>1.0.0-SLEUTH</version>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.4.4</version>
```
Comment (jonatan-ivanov) [inline config-server/pom.xml:12]: How about 2.4.5?

File: config-server/src/main/java/com/example/sleuthsamples/ConfigServerApplication.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		try (Tracer.SpanInScope spanInScope = this.tracer.withSpan(hello.start())) {
+			int port = environment.getProperty("server.port", Integer.class, 8888);
+			this.restTemplate.getForObject("http://localhost:" + port + "/master/application.yml", String.class);
+
+		} finally {
+			hello.end();
+		}
+
+	}
+}
```
Comment (jonatan-ivanov) [inline config-server/src/main/java/com/example/sleuthsamples/ConfigServerApplication.java:70]: There are a few files that does not have newline at the end. Do we have `.editorconfig` here?

File: config-server/src/main/resources/application.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+server.port: 8888
+spring.cloud.config.server.git.uri: https://github.com/spring-cloud-samples/config-repo
+
+spring:
+  application:
+    name: config-server
```
Comment (jonatan-ivanov) [inline config-server/src/main/resources/application.yaml:6]:
```suggestion
spring.application.name: config-server
```

File: config-server/.mvn/wrapper/maven-wrapper.properties (hunk 1 above)
Comment (marcingrzejszczak) [reply config-server/.mvn/wrapper/maven-wrapper.properties:1]: We can do a bump of all projects at the same time

File: config-server/pom.xml (hunk 2 above)
Comment (marcingrzejszczak) [reply config-server/pom.xml:12]: We can do a bump of all projects at the same time in a separate commit

**Review by jonatan-ivanov — APPROVED** [review]

**Comment by marcingrzejszczak** [comment]
done via https://github.com/spring-cloud-samples/spring-cloud-sleuth-samples/commit/ad80f7bf369214ae4dd9c6a7185f43f6ceade0cc

**Pull request closed without merging**